powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -142,7 +141,7 @@ public boolean createDevice(Long appId, String token, MobilePlatform platform, S
         request.setActivationId(activationId);
         TypeReference<Response> typeReference = new TypeReference<Response>() {
         };
-        ObjectResponse<?> response = postObjectImpl("/push/device/create", new ObjectRequest<>(request), typeReference);
+        ObjectResponse<?> response = postObjectImpl("/push/device/create", request, typeReference);
```

Comment:
Oh boy... Why do you make changes like this? :) Why is this supposed to continue to work, after you arbitrarily change the request object type? The change is not correct, and I have noticed that even the new methods that you implemented (delete, update status, ...) have the same issue and use incorrect request object...

This method (and all other) must always call:

```java
new ObjectRequest<>(request)
```

Try to figure out why is that...